src/components/products/product-subscribe/ko/runtime/product-subscribe.html

```diff
@@ -22,11 +22,11 @@
 
 <!-- ko ifnot: limitReached -->
 <form>
-    <div class="table">        
+    <div class="table">
         <div class="table-row">
             <!-- ko ifnot: delegationEnabled -->
-            <div class="col-5">
-                <input type="text" class="form-control" placeholder="Your new product subscription name" data-bind="textInput: subscriptionName" />
+            <div class="col-5 form-inline">
```

Comment:
We don't need to change styles of the existing experience. Not sure how all existing customer would react so let's not do this.